katas/content/superposition/greenberger_horne_zeilinger/solution.md

```diff
@@ -0,0 +1,26 @@
+### Solution
+
+The single-qubit GHZ state is the plus state $\frac{1}{\sqrt{2}} \big (|0\rangle + |1\rangle\big)$ that we've discussed in task 1.1. As a reminder, that state is prepared by applying a Hadamard gate.
```

Comment:
The tasks are not numbered in the new interface (you can see examples at https://quantum.microsoft.com/en-us/experience/quantum-katas - they just have titles), so we need to refer to them in some other way. Here, we can probably just say "in the first task", since that doesn't force the learner to count much